### DISCUSSION TITLE
Icon for indicator plugin (XFCE) with appimage version

### DISCUSSION THREAD

**Discussion by JNC92** [keepassxreboot/keepassxc#8753] [Q&A]
Hi,

Newbie in GitHub, just signed up to ask my question here about the appimage version:
KeePassXC - Version 2.7.4
Révision : 63b2394
Distribution : AppImage

I use XFCE on my xubuntu 22.10 machine and the icon for the application in the indicator plugin is wrong, see screenshot.
Notes:

- It was fine with the version bundled with 22.10 (2.6.6). Meaning: correct KeepassXC icon displayed in the panel.
- The icon for another appimage application (Rambox) is fine (the 2nd below KeepassXC).
- Plugin: xfce4-indicator-plugin - Indicator

For my main menu, I have done the same as for Rambox: put the svg version of the icon in the same directory than the appimage file (in /opt). But it doesn't seem to be the same icon anyway (the Rambox icon is different in the main menu and in the indicator plugin).
I haven't found anything on the indicator plugin end to manually add an icon. If Rambox managed to do it, I guess that it should be doable in the appimage package.
Thanks!

![KeepassXC](https://user-images.githubusercontent.com/117534535/200132652-85c51324-4713-47ba-bb53-7b32882a0473.png)

**Comment by JNC92** [comment]
Edit
For the record, I tried with the flatpak version (2.7.4 too) and it works fine. Had tried without restarting after installing flatpak.

**Reply by droidmonkey** [reply]
This is very likely a similar problem to opening links with appimage.

**Reply by JNC92** [reply]
Thanks. A pity. This Appimage is was quite good IMHO.
